features/dev/persoon-adressering-inonderzoek.feature

```diff
@@ -339,10 +339,10 @@ Functionaliteit: Persoon adressering in onderzoek
       | aanduiding in onderzoek (83.10) | <gba in onderzoek waarde> |
       | datum ingang onderzoek (83.20)  | 20020701                  |
       En de 'verblijfplaats' heeft de volgende 'adres' gegevens
-      | naam                              | waarde |
-      | gemeente van inschrijving (09.10) | 0518   |
-      | straatnaam (11.10)                | Spruit |
-      | huisnummer (11.20)                | 123    |
+      | naam                 | waarde |
+      | gemeentecode (92.10) | 0518   |
```

Comment:
Kan jij deze wijziging toelichten? Verblijfplaats kent geen gemeentecode en ook geen 92.10 rubriek. Als we het hebben over een code van een gemeente binnen verblijfplaats dan moet dat gemeente van inschrijving (09.10) zoals het hiervoor stond.

Reply:
Dit is de gemeente_code kolom voor lo3_adres tabel. Dit is een verplicht veld voor die tabel